lib/services/security/checkpoint.js

```diff
@@ -112,6 +112,7 @@ CheckPoint.prototype.__loadPermissionSet = function(identity, callback) {
     (groupName, eachCB) => {
       this.securityService.groups.getGroup(groupName, {}, (e, group) => {
         if (e) return eachCB(e);
+        if (group == null) return eachCB();

```

Comment:
Does this not need an error with the callback?

Reply:
No - this is a state thing - the group may have been deleted as we are loading the permission set - so this is a valid condition.